app/dashboard/migrations/0050_auto_20190829_2141.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.3 on 2019-08-29 21:41
```

Comment:
Can we get the migrations in this PR squashed down?

Reply:
sure thang. .was waiting to see which migrations get into master/prod this release first so i could base it off of that / not

Reply:
squashed! 5f85efe31defc430244f02d9d7a66bf2ab2ac74a